Zotero/Scenes/Detail/Items/Views/ItemsFilterViewController.swift

```diff
@@ -61,7 +61,7 @@ class ItemsFilterViewController: UIViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
 
-        guard UIDevice.current.userInterfaceIdiom == .pad else { return }
+        guard traitCollection.horizontalSizeClass == .regular && UIDevice.current.userInterfaceIdiom == .pad else { return }
```

Comment:
Is `viewWillAppear` called when size class changes? If not then the `preferredSize` should be updated on size class change.